docs/guide/docs/infrastructure/database.md

```diff
@@ -1,27 +1,27 @@
 ---
 id: database
-title: Supported Databases
+title: Supported databases
 ---
 
-Botpress comes with support for SQL databases out-the-box and can be accessed by:
+Botpress comes with support for SQL databases out-the-box. These can be accessed using the following methods:
```

Comment:
can we just remove that section ? from line 6 to 19 I'd like if we simply remove that as I feel it's more confusing than valuable

Reply:
Okay I agree, let me fix that.